gravitee-plugin-repository/src/main/java/io/gravitee/plugin/repository/internal/RepositoryPluginHandler.java

```diff
@@ -211,7 +211,19 @@ private void checkRepositoryConfig(final Scope scope, final boolean failOnMissin
     }
 
     private String getRepositoryType(Scope scope) {
-        return environment.getProperty(scope.getName() + ".type");
+        String repositoryType = environment.getProperty("repositories." + scope.getName() + ".type");
```

Comment:
Would you be able to add dedicated unit test for this new configuration ?

Reply:
Sure, I extracted the relevant logic to a separate class and tested it.